TestFramework/Tests.Integration/Tests/UserIntegrationTests.cs

```diff
@@ -323,7 +323,8 @@ public async Task TestLoginSalesforceAsyncBad()
 		// MIC LOGIN TESTS
 		//
 		[Test]
-		public void TestMIC_LoginWithMIC_NormalFlow()
+        [Obsolete("This test has been deprecated.")]
+        public void TestMIC_LoginWithMIC_NormalFlow()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?